stats-server/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "cargo-quickinstall-stats-server"
+version = "0.0.0"
+authors = ["David Laban <alsuren@gmail.com>"]
+# based on https://github.com/fly-apps/hello-rust by Jerome Gravel-Niquet <jeromegn@gmail.com>
+edition = "2021"
+
+# Make a new workspace so that we get our own Cargo.lock and target dir for the docker build. 
+[workspace]
```

Comment:
For some reason my vscode is no longer giving any hints for anything in this folder unless I explicitly open only this folder in its own instance. I'm sure it worked before, and for other projects, but I can't remember what I did last time.

Probably I will merge this as is, but if this causes problems for anyone else then tell me and I'll try to address it properly.